src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php

```diff
@@ -93,9 +89,7 @@ public function getPretty(): string
 
     public function getInfo(string $eventName): array
     {
-        if (null === $this->stub) {
-            $this->stub = self::$hasClassStub ? new ClassStub($this->pretty.'()', $this->listener) : $this->pretty.'()';
-        }
+        $this->stub ??= self::$hasClassStub ? new ClassStub($this->pretty.'()', $this->listener) : $this->pretty.'()';
```

Comment:
this change can be skipped

Reply:
It cannot, because $this->stub might be uninitialized. In that case, `null === $this->stub` will error.